appcues/src/main/java/com/appcues/ui/theme/AppcuesColors.kt

```diff
@@ -1,23 +0,0 @@
-package com.appcues.ui.theme
-
-import androidx.compose.ui.graphics.Color
-
-internal object AppcuesColors {
```

Comment:
Deleted since accessing colors directly made it impossible to switch between light and dark mode